packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx

```diff
@@ -42,30 +42,13 @@ const StyledScrollableNav = styled.div`
   }
 `;
 
-export const CanonicalScrollableNavigation = ({
-  children,
-  dir,
-  isScrollable,
-}) => {
-  const ariaHidden = isScrollable && { 'aria-hidden': true };
-
-  return (
-    <StyledScrollableNav dir={dir} {...ariaHidden}>
-      {React.Children.map(children, child =>
-        React.cloneElement(child, { isScrollable }),
-      )}
-    </StyledScrollableNav>
-  );
-};
+export const CanonicalScrollableNavigation = ({ children, dir }) => (
```

Comment:
This is now exactly the same as the Amp scrollable nav right? And we probably need the spread props for canonical too for event tracking? Should we just merge them?